AttributeError: 'dict_keys' object has no attribute 'sort'

I am using `MacFSEvents` with [sniffer](https://github.com/jeffh/sniffer). It seems that the recent `0.5` release caused a regression in Python 3.5:

```
env/bin/sniffer
Traceback (most recent call last):
  File "env/bin/sniffer", line 7, in <module>
    from sniffer import main
  File "<...>/env/lib/python3.5/site-packages/sniffer/__init__.py", line 26, in <module>
    from .scanner import Scanner
  File "<...>/env/lib/python3.5/site-packages/sniffer/scanner/__init__.py", line 34, in <module>
    _import('fsevents_scanner', 'FSEventsScanner')    # osx
  File "<...>/env/lib/python3.5/site-packages/sniffer/scanner/__init__.py", line 28, in _import
    mod = __import__(str(module), globals(), locals(), [cls], 1)
  File "<...>/env/lib/python3.5/site-packages/sniffer/scanner/fsevents_scanner.py", line 9, in <module>
    import fsevents
  File "<...>/env/lib/python3.5/site-packages/fsevents.py", line 47, in <module>
    class Mask(int):
  File "<...>/env/lib/python3.5/site-packages/fsevents.py", line 72, in Mask
    svals.sort()
AttributeError: 'dict_keys' object has no attribute 'sort'
make: *** [watch] Error 1
```

If I specifically install the previous release (`$ pip install MacFSEvents==0.4`), I do not see this issue.

--- a/fsevents.py
+++ b/fsevents.py
@@ -45,7 +45,6 @@ from _fsevents import (
     )
 
 class Mask(int):
-    __slots__ = ()
 
     stringmap = {
         FS_FLAGMUSTSCANSUBDIRS:    'MustScanSubDirs',

